flink/v1.12/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -364,7 +364,7 @@ private String operatorName(String suffix) {
 
       // Fallback to use upsert mode parsed from table properties if don't specify in job level.
       boolean upsertMode = upsert || PropertyUtil.propertyAsBoolean(table.properties(),
-          UPSERT_MODE_ENABLE, UPSERT_MODE_ENABLE_DEFAULT);
+          UPSERT_ENABLED, UPSERT_ENABLED_DEFAULT);
```

Comment:
Do we need a corresponding change for the job level check? The one that the variable `upsert` comes from?

Reply:
I think you mean it's also good to name [those variables](https://github.com/apache/iceberg/blob/master/flink/v1.12/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java#L225-L229) to `enabled`.

Reply:
I renamed, @openinx.